<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TextureSelector</class>
 <widget class="QWidget" name="TextureSelector">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>354</width>
    <height>205</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>2</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string/>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3" stretch="0">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4" stretch="1,0">
          <item>
           <widget class="QLabel" name="texTypeLabel">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="deleteTextureSelector">
            <property name="maximumSize">
             <size>
              <width>24</width>
              <height>16777215</height>
             </size>
            </property>
            <property name="text">
             <string>X</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6" stretch="0,0">
          <item>
           <widget class="QLabel" name="texSurf">
            <property name="minimumSize">
             <size>
              <width>150</width>
              <height>150</height>
             </size>
            </property>
            <property name="maximumSize">
             <size>
              <width>150</width>
              <height>150</height>
             </size>
            </property>
            <property name="styleSheet">
             <string notr="true"/>
            </property>
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <layout class="QVBoxLayout" name="verticalLayout_4" stretch="1,0,0,0">
            <item>
             <widget class="QPushButton" name="texSelect">
              <property name="text">
               <string>Select</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout" stretch="0,0">
              <item>
               <widget class="QLabel" name="label">
                <property name="text">
                 <string>Sampler:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="samplerBox">
                <item>
                 <property name="text">
                  <string>Diffuse</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Specular</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Normal</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <item>
               <widget class="QLabel" name="label_2">
                <property name="text">
                 <string>Min Filter</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox">
                <item>
                 <property name="text">
                  <string>Linear</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Nearest</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3" stretch="0,0">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="text">
                 <string>Mag Filter:</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QComboBox" name="comboBox_2">
                <item>
                 <property name="text">
                  <string>Linear</string>
                 </property>
                </item>
                <item>
                 <property name="text">
                  <string>Nearest</string>
                 </property>
                </item>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
